Wolox/wlogger-ios
# local test coverage enabled

## Sumary ##
**Trello card**: https://trello.com/c/c3SX0f1t/15-integrar-con-coveralls

Test coverage enabled and Run command runs and tests

## Test Plan ##
  
* Run the tests
* After they finish running, tap the bubble icon, which is the 8th icon above the project groups
* Click the last build, which is the one you've just run, and select "Coverage" in the top-middle.
* Verify that the coverage is displayed
